include/zrepl_mgmt.h

```diff
@@ -36,7 +37,7 @@ extern "C" {
 #define	uZFS_ZVOL_WORKERS_MAX 128
 #define	uZFS_ZVOL_WORKERS_DEFAULT 6
 
-extern pthread_mutex_t zvol_list_mutex;
```

Comment:
this should be declared with extern and the actual definition should be in uzfs_mgmt.c. Otherwise every file which includes this header file will have a global variable of this name.